### PR TITLE
Test Noobaa resources restart

### PR DESCRIPTION
**Pull request by ebenahar**

### CODE REVIEW

File: tests/manage/mcg/test_restart_noobaa_resources.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        """
+        labels_map = {
+            'noobaa_core': constants.NOOBAA_CORE_POD_LABEL,
+            'noobaa_db': constants.NOOBAA_DB_LABEL
+        }
+        pod_obj = self.resource_obj = pod.Pod(
+            **pod.get_pods_having_label(
+                label=labels_map[resource_to_delete],
+                namespace=defaults.ROOK_CLUSTER_NAMESPACE
```
Comment (MeridianExplorer) [inline tests/manage/mcg/test_restart_noobaa_resources.py:50]: Might be better to use `config.ENV_DATA['cluster_namespace']`

**Comment by MeridianExplorer** [comment]
Thanks a lot!

File: tests/manage/mcg/test_restart_noobaa_resources.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        pytest.param(
+            *['noobaa_core'], marks=pytest.mark.polarion_id("OCS-2232")
+        ),
+        pytest.param(
+            *['noobaa_db'], marks=pytest.mark.polarion_id("OCS-2233")
+        )
+    ]
+)
+@pytest.mark.usefixtures(setup.__name__)
+class TestRestartNoobaaResources(ManageTest):
```
Comment (RazTamir) [inline tests/manage/mcg/test_restart_noobaa_resources.py:34]: do we plan to add more test cases with IO in the bg?
LGTM

File: tests/manage/mcg/test_restart_noobaa_resources.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import logging
```
Comment (AviadP) [inline tests/manage/mcg/test_restart_noobaa_resources.py:1]: blank line between 3rd party and built-in modules

**Pull request merged**